sections/announcement-bar.liquid

```diff
@@ -1,18 +1,33 @@
 {{ 'component-slideshow.css' | asset_url | stylesheet_tag }}
 {{ 'component-slider.css' | asset_url | stylesheet_tag }}
+{%- if section.settings.enable_country_selector or section.settings.enable_language_selector -%}
+  {{ 'component-localization-form.css' | asset_url | stylesheet_tag: preload: true }}
+{%- endif -%}
+
+{%- if section.settings.enable_country_selector or section.settings.enable_language_selector -%}
+  <script src="{{ 'localization-form.js' | asset_url }}" defer="defer"></script>
+{%- endif -%}
 
 {%- liquid
   assign social_icons = true
   if settings.social_facebook_link == blank and settings.social_instagram_link == blank and settings.social_youtube_link == blank and settings.social_tiktok_link == blank and settings.social_twitter_link == blank and settings.social_pinterest_link == blank and settings.social_snapchat_link == blank and settings.social_tumblr_link == blank and settings.social_vimeo_link == blank
     assign social_icons = false
   endif
+  assign language_country_selector = false
+  if section.settings.enable_country_selector or section.settings.enable_language_selector
+    assign language_country_selector = true
+  endif
+  assign announcement_bar = true
+  if section.blocks.size == 0
+    assign announcement_bar = false
+  endif
```

Comment:
Same here, only worth changing if we're making other more important changes anyway, but this probably be slightly shortened to just...
```
if section.blocks.size > 0
  assign announcement_bar = true
endif
```

Even if we kept both assigns for false and true, I'd flip the logic so it's false by default, true in some condition. (like we have just above)